site/pages/sysadmin/manage-organisations.page.tsx

```diff
@@ -79,11 +79,13 @@ const ManageOrganisations = ({ orgList, csrfToken }: ManageOrganisationsProps):
                     ]}
                 />
             ) : null}
-            <Table
-                caption={{ text: "Manage organisations", size: "m" }}
-                columns={["Organisation", "NaPTAN AdminArea", "Action"]}
-                rows={getRows()}
-            ></Table>
+            <h1 className="govuk-heading-l">Manage organisations</h1>
+            <p className="govuk-body">
+                Create, edit and remove organisations who can use the CTDD tool. Assign NaPTAN AdminArea&apos;s per
```

Comment:
I know it's in the designs but there shouldn't be an apostrophe here
```suggestion
                Create, edit and remove organisations who can use the CTDD tool. Assign NaPTAN AdminAreas per
```

Reply:
done